pkg/handler/scale_jobs.go

```diff
@@ -0,0 +1,70 @@
+package handler
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	keda_v1alpha1 "github.com/kedacore/keda/pkg/apis/keda/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (h *ScaleHandler) scaleJobs(scaledObject *keda_v1alpha1.ScaledObject, isActive bool, scaleTo int64, maxScale int64) {
+	// TODO: get current job count
+	log.Println("Scaling Jobs")
```

Comment:
what log level does this end up being? might be better to explicitly give it `info` level

Reply:
It doesn't end up being one, it just becomes a thread safe fmt.Println, but I've switched them to Info level now. Thanks :)